src/diffusers/models/attention_processor.py

```diff
@@ -2751,6 +2783,117 @@ def __call__(
         return hidden_states
 
 
+class XLAFlashAttnProcessor2_0:
+    r"""
+    Processor for implementing scaled dot-product attention with pallas flash attention kernel if using `torch_xla`.
+    """
+
+    def __init__(self, partition_spec: Optional[Tuple[Optional[str], ...]] = None):
+        if not hasattr(F, "scaled_dot_product_attention"):
+            raise ImportError("XLAFlashAttnProcessor2_0 requires PyTorch 2.0, to use it, please upgrade PyTorch to 2.0.")
```

Comment:
```suggestion
```
I think we don't need to support SDPA in XLA Flash attention processor! let's remove all the logics related to that to simplify things a bit!

Reply:
Please check my comment above for why we still keep it here. Thanks